setup.py

```diff
@@ -1,11 +1,15 @@
 from setuptools import setup
 
+with open('requirements.txt') as f:
+    install_requires = f.read().strip().split('\n')
```

Comment:
I personally prefer to keep it all in the setup.py file, but this isn't a strong preference.

Reply:
Most projects today seem to keep things in requirements.txt .  I'm inclined to stay with that convention.

I also like this personally because it's easy to look up what a package requires quickly in a data file rather than looking through a setup.py file.  Although in this case the setup.py is simple enough that this isn't a very solid argument.  The "common convention" one though I think does stand.